### PR TITLE
documentation -> docs

### PR DESCRIPTION
**Pull request by dhrubabasu**

### CODE REVIEW

File: docusaurus.config.js [hunk 1]
```diff
@@ -6,9 +6,9 @@ const darkCodeTheme = require('prism-react-renderer/themes/dracula');
 
 /** @type {import('@docusaurus/types').Config} */
 const config = {
-  title: 'Developer Documentation - Avalanche',
-  tagline: 'Avalanche Developer Documentation',
-  url: 'https://docs.avax.network',
+  title: 'Avalanche',
```
Comment (patrick-ogrady) [inline docusaurus.config.js:9]: Why not Avalanche Docs here?

Comment (patrick-ogrady) [reply docusaurus.config.js:9]: Seems like a bad title

**Review by yulin-dong — APPROVED** [review]

File: docusaurus.config.js [hunk 2]
```diff
@@ -6,9 +6,9 @@ const darkCodeTheme = require('prism-react-renderer/themes/dracula');
 
 /** @type {import('@docusaurus/types').Config} */
 const config = {
-  title: 'Developer Documentation - Avalanche',
-  tagline: 'Avalanche Developer Documentation',
-  url: 'https://docs.avax.network',
+  title: 'Avalanche Docs',
+  tagline: 'Documentation and guides for Avalanche',
```
Comment (patrick-ogrady) [inline docusaurus.config.js:10]: Documentation and Tutorials for Avalanche

Comment (yulin-dong) [reply docusaurus.config.js:10]: https://github.com/ava-labs/avalanche-docs-beta/commit/47b6e2587bb4b0270d9ac33bc131370d68d94022

**Pull request merged**